articles/overview/understanding-quantum-computing.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Understanding quantum computing
+description: What are quantum computers and how do they use the principles of quantum mechanics?
+author: bradben
+ms.author:  bradben
+ms.date: 5/5/2020
+ms.topic: overview
+uid: microsoft.quantum.overview.understanding
+---
+
+# Understanding quantum computing
+
+Quantum computing uses the principles of quantum mechanics to process information. Because of this, quantum computing requires a different approach than classical computing.  One example of this difference is the processor used in quantum computers.  Where classical computers use familiar silicon-based chips, quantum computers use quantum material such as atoms, ions, photons, or electrons.  
```

Comment:
```suggestion
Quantum computing uses the principles of quantum mechanics to process information. Because of this, quantum computing requires a different approach than classical computing.  One example of this difference is the processor used in quantum computers.  Where classical computers use familiar silicon-based chips, quantum computers use quantum materials such as atoms, ions, photons, or electrons.  
```